service/src/main/java/bio/terra/axonserver/app/controller/AwsResourceController.java

```diff
@@ -100,7 +101,25 @@ public AwsSageMakerNotebook getNotebook(UUID workspaceId, UUID resourceId) {
     checkAwsEnabled();
     String accessToken = getAccessToken();
     return AwsSageMakerNotebook.create(
-        wsmService, wsmService.getResource(workspaceId, resourceId, accessToken), accessToken);
+        wsmService,
+        wsmService.getResource(workspaceId, resourceId, accessToken),
+        /*awsCredentialAccessScope=*/ null,
+        accessToken);
+  }
+
+  /** Do not use, public for spy testing */
+  @VisibleForTesting
+  public AwsSageMakerNotebook getNotebook(
```

Comment:
@rogerwangcs Could you mark this visible only within the package (just remove the access specifier)? it is visible to tests in the same package

Reply:
This method is only used in this class + its test. Do you need this change for something?

Reply:
Since it is used only in its class & test, you can simply remove the public access specifier
reason - general guideline: provide least access level that is necessary. 
this is NOT urgent and can be done in the next PR on this repo.